### PR TITLE
Adds job definition for visual regression

### PR DESCRIPTION
**Pull request by MuriloDalRi**
Ticket: https://trello.com/c/y9YBmNH6/41-mvp-implement-visual-regression

### CODE REVIEW

File: job_definitions/release_pipeline.yml [hunk 1]
```diff
@@ -59,6 +61,81 @@
...
+                    echo "Approving visual regression tests failed: ${inside_error}"
+                    return false
+                  }
+
+                  return true
+                } else {
+                  input(message: "Visual regression tests have run elsewhere since this pipeline triggered the job. They will be re-run here when you proceed.")
+                  return false
+                }
+              } else if (env.FAILED_TEST_ACTION == 'Bypass') {
```
Comment (Wynndow) [inline job_definitions/release_pipeline.yml]: When would this step every be useful?

File: job_definitions/release_pipeline.yml [hunk 2]
```diff
@@ -59,6 +61,81 @@
...
+                    ]
+                  ]
+                }
+
+                echo "Action chosen was ${env.FAILED_TEST_ACTION}"
+              }
+
+              if (env.FAILED_TEST_ACTION == 'Retry') {
+                echo "Retrying visual regression tests on ${stage}."
+                return false
```
[the remark this answers is not in the document]
Comment (samuelhwilliams) [reply job_definitions/release_pipeline.yml]: Yep, it kicks back to the start of the waitUntil loop.

File: job_definitions/release_pipeline.yml (hunk 1 above)
Comment (samuelhwilliams) [reply job_definitions/release_pipeline.yml]: I don't have a specific use case in mind, but we have the same option on the functional tests, so presumably there's a use case somewhere.

Comment (Wynndow) [reply job_definitions/release_pipeline.yml]: Yeah, I guess having the flexibility is a good thing!

Comment (samuelhwilliams) [reply job_definitions/release_pipeline.yml]: One possible example: someone's testing a framework in a different state (e.g. g-cloud 8 open), and the only changes failing a report are from the banners that appear on the homepage/dashboards. In this case, the framework might be open for an indeterminate amount of time, so you can't really wait to re-run them, but you can't approve the tests either. Bypassing them at that point might be OK as long as the developer was sure no unintended changes slipped through.

**Pull request merged**